virttest/qemu_vm.py

```diff
@@ -1459,7 +1459,8 @@ def add_disable_modern(devices, dev, dev_type):
                     if params.get('machine_type').startswith("arm64-mmio"):
                         dev = QDevice('virtio-serial-device')
                     else:
-                        dev = QDevice('virtio-serial-pci', parent_bus=pci_bus)
+                        parent_bus = {"aobject": port_params.get("port_bus", "pci.0")}
```

Comment:
Also there is a virtio_serial_port added in `add_log_anaconda`. Shouldn't that get the same treatment?